Makefile

```diff
@@ -128,14 +128,18 @@ $(BINARY_DIR)/$(TEST_BINARY_NAME): $(BINARY_DIR) $(SRCS)
 
 ##@ Setup
 
+.PHONY: install-dep
```

Comment:
We have to install dep first in the separated target so that `$(eval GINKGO_VERSION:=$(shell dep status -f='{{if eq .ProjectRoot "github.com/onsi/ginkgo"}}{{.Version}}{{end}}'))` will work in the `tools` target. Otherwise `dep` is not on path yet.